itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/cache/TestCachedStoreUpdateUsingEvents.java

```diff
@@ -94,8 +96,9 @@ private Database createTestDb(String dbName, String dbOwner) {
   }
 
   private Table createTestTblParam(String dbName, String tblName, String tblOwner,
-                              List<FieldSchema> cols, List<FieldSchema> ptnCols, Map<String, String> tblParams) {
-    String serdeLocation = "file:/tmp";
+                              List<FieldSchema> cols, List<FieldSchema> ptnCols, Map<String, String> tblParams)
+      throws IOException {
+    String serdeLocation = Files.createTempDirectory(dbName + "_" + tblName).toString();
```

Comment:
Even the dbName and tblName can be same across multiple tests, which can get cleaned up. Would it be a good idea to use the current timestamp or generate a random number and use that for the temporary directory ?

Reply:
same comment as above. Based on the documentation of Files.createTempDirectory it returns "the path to the newly created directory that did not exist before this method was invoked"